### PR TITLE
Update releases.toml.

### PR DESCRIPTION
**Pull request by DmitryLukyanov**
Doing it explicitly to ensure that nothing missed in my first releasing

### CODE REVIEW

File: Docs/landing/data/releases.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 current = "2.12.1"
```
Comment (JamesKovacs) [inline Docs/landing/data/releases.toml:1]: Need to update current.

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Missed one.

**Comment by JamesKovacs** [comment]
I would recommend doing the PR on `master` and then cherrypicking to `v2.12.x`.

File: Docs/landing/data/releases.toml (hunk 1 above)
Comment (DmitryLukyanov) [reply Docs/landing/data/releases.toml]: done

**Comment by DmitryLukyanov** [comment]
moved this code into different PR for the master

**Pull request closed without merging**